src/main/thrift/tag.thrift

```diff
@@ -153,4 +155,14 @@ struct Tag {
 
     /** The time at which this tag was last updated */
     22: optional i64 updatedAt;
+
+    /** The currently active sponsorships for this tag */
+    23: optional list<sponsorship.Sponsorship> activeSponsorships;
+
+    /** the id of the sponsorship for a paid content tag, for internal use
+    external consumers should only use the activeSponsorships data */
+    25: optional i64 sponsorshipId;
+
+    /** is this tag expired - only applies to paid content tags */
+    26: required bool expired;
```

Comment:
It'll only be set to true for paid content tags, all other tags should assume expired = false. Since this flag will always be set to something in CPI I thought making it required and explicit was not a bad thing.